src/olympia/reviews/tests/test_models.py

```diff
@@ -106,6 +132,93 @@ def test_no_filter_for_relations(self):
         flag = ReviewFlag.objects.get(pk=flag.pk)
         assert flag.review == review
 
+    def test_creation_triggers_email_and_logging(self):
+        addon = Addon.objects.get(pk=4)
+        addon_author = user_factory()
+        addon.addonuser_set.create(user=addon_author)
+        review_user = user_factory()
+        review = Review.objects.create(
+            user=review_user, addon=addon,
+            body=u'Rêviiiiiiew', user_responsible=review_user)
+
+        activity_log = ActivityLog.objects.latest('pk')
+        assert activity_log.user == review_user
+        assert activity_log.arguments == [addon, review]
+        assert activity_log.action == amo.LOG.ADD_REVIEW.id
+
+        assert len(mail.outbox) == 1
+        email = mail.outbox[0]
+        reply_url = helpers.absolutify(
+            helpers.url('addons.reviews.reply', addon.slug,
+                        review.pk, add_prefix=False))
+        assert email.subject == 'Mozilla Add-on User Review: my addon name'
+        assert 'A user has left a review for your add-on,' in email.body
+        assert 'my addon name' in email.body
+        assert reply_url in email.body
+        assert email.to == [addon_author.email]
+        assert email.from_email == 'Mozilla Add-ons <nobody@mozilla.org>'
+
+    def test_reply_triggers_email_but_no_logging(self):
```

Comment:
Just an observation, and I know we don't currently, but _shouldn't_ a reply leave a log entry?